src/graphics/rectangle.rs

```diff
@@ -27,4 +27,15 @@ impl Rectangle<f32> {
             && self.y <= point.y
             && point.y <= self.y + self.height
     }
+
+    /// Returns [`Point`] that is exactly in the center of this [`Rectangle`].
+    ///
+    /// [`Point`]: type.Point.html
+    /// [`Rectangle`]: struct.Rectangle.html
+    pub fn center(&self) -> Point {
+        Point::new(
+            (self.x + self.width) / 2.0, 
+            (self.y + self.height) / 2.0,
```

Comment:
I think this should be `self.x + self.width / 2.0`.

Reply:
oh, you're absolutely right!